jenkins/hourly_maintenance.py

```diff
@@ -137,13 +140,41 @@ def KillLoopingBash():
             err |= subprocess.call(['sudo', 'kill', '-9', pid])
     return err
 
+
+def DeleteCorruptGitRepos():
```

Comment:
yeah, I'm mostly just worried about the hour or so between agent crashing and this script running.